frontend/sw360-portlet/src/main/resources/META-INF/resources/html/utils/includes/editAdditionalData.jsp

```diff
@@ -17,6 +17,49 @@
             <th colspan="3" class="headlabel"><liferay-ui:message key="additional.data" /></th>
         </tr>
     </thead>
+	<c:forEach var="customField" items="${customFields}">
+		<tr <c:if test="${customField.hidden}">style="display:none"</c:if> class="bodyRow">
+			<td>
+				<div>
+					<input class="form-control" id="${customField.fieldKey}" name="<portlet:namespace/><%=PortalConstants.ADDITIONAL_DATA_KEY%>${customField.fieldKey}"
+						   type="text" value="<sw360:out value="${customField.fieldLabel}"/>" readonly/>
+				</div>
+			</td>
+			<td>
+				<div>
+					<c:if test="${customField.fieldType == 'TEXTFIELD'}">
+						<input class="form-control" id="${customField.fieldLabel}" name="<portlet:namespace/><%=PortalConstants.ADDITIONAL_DATA_VALUE%>${customField.fieldKey}"
+							   type="text" placeholder="Enter ${customField.fieldLabel}"
+							   <c:if test="${not empty customField.fieldPattern}">pattern="${customField.fieldPattern}" title="${customField.fieldPattern}"</c:if>
+							   value="<sw360:out value="${customField.value}"/>"/>
+					</c:if>
+					<c:if test="${customField.fieldType == 'TEXTAREA'}">
+						<textarea class="form-control" id="${customField.fieldLabel}" name="<portlet:namespace/><%=PortalConstants.ADDITIONAL_DATA_VALUE%>${customField.fieldKey}"
+								  rows="4" placeholder="Enter ${customField.fieldLabel}">${customField.value}</textarea>
+					</c:if>
+					<c:if test="${customField.fieldType == 'DATE'}">
+						<input id="${customField.fieldLabel}" class="datepicker form-control" name="<portlet:namespace/><%=PortalConstants.ADDITIONAL_DATA_VALUE%>${customField.fieldKey}" type="text"
+							   placeholder="Enter ${customField.fieldLabel}" pattern="\d{4}-\d{2}-\d{2}"
+							   value="<sw360:out value="${customField.value}"/>"/>
+					</c:if>
+					<c:if test="${customField.fieldType == 'DROPDOWN'}">
+						<select class="form-control" id="${customField.fieldLabel}" name="<portlet:namespace/><%=PortalConstants.ADDITIONAL_DATA_VALUE%>${customField.fieldKey}">
+							<option value="">Select:</option>
+							<c:forEach var="option" items="${customField.options}">
+								<option value="${option}" <c:if test="${option eq customField.value}">selected</c:if>>${option}</option>
+							</c:forEach>
+						</select>
+					</c:if>
+					<c:if test="${customField.fieldType == 'CHECKBOX'}">
+						<input id="${customField.fieldLabel}" type="checkbox" class="form-check-input" name="<portlet:namespace/><%=PortalConstants.ADDITIONAL_DATA_VALUE%>${customField.fieldKey}"
```

Comment:
Yes, multi-checkbox is not supported, because of the addtionalData is key:value string pair.
Following is a workaround to store multi values.
Component Status 1: Checkbox with value 1
Component Status 2: Checkbox with value 2